lte/gateway/docker/s1ap/readme.md

```diff
@@ -71,3 +45,35 @@ If inside of container, CTRL+d or exit from container and run stop script
 root@472f8708ec12:/magma/lte/gateway/python/integ_tests# exit
 s1ap/stop-s1ap.sh
 ```
+
+# Simple example
+
+```
+sudo -i
+wget https://raw.githubusercontent.com/magma/magma/master/lte/gateway/deploy/agw_install_docker.sh
+mkdir -p /var/opt/magma/certs
+echo "-----BEGIN CERTIFICATE-----
+MIIDXzCCAkegAwIBAgIUakfCUNf7JMKbLDqHnuiG1QNhCQ8wDQYJKoZIhvcNAQEL
+BQAwPzELMAkGA1UEBhMCVVMxMDAuBgNVBAMMJ3Jvb3RjYS5tYWdtYS0xNi1zeWRu
+ZXkuZmFpbGVkd2l6YXJkLmRldjAeFw0yMTA3MTQyMjI5MjNaFw0zMTA3MTIyMjI5
+MjNaMD8xCzAJBgNVBAYTAlVTMTAwLgYDVQQDDCdyb290Y2EubWFnbWEtMTYtc3lk
+bmV5LmZhaWxlZHdpemFyZC5kZXYwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEK
+AoIBAQDFVmNFaAOkVD3c8W28FkGUVmBKDyj/T7N8C7PE43WvbBZJmO5TO1c887Dt
+yiX8Ua2mpCQ2SF8DZtXojkLGKOFM85uxzTV1YI656u5BDSejRkm1UDeMT5R+tQJK
+fyHYTt5ZNprX/dUrxYnp+h2zEl0PlzO5ijrktuZgM4KZjtQVaC1VirSC//2ZKQEo
+2aX3L81ALrjVzsmH4TePKEY8StjDHC2Mg6LOaYR/+Gu272P39/heULrm147g1k0k
+haeKv8qrI0dfvBcZveTzYf77iA6/OeVzYtWwM3zr1Z1cFALZrcuS6R6DrsAInseH
+qeiMh4kLfyoh0vQCNpEAJQgt5PmVAgMBAAGjUzBRMB0GA1UdDgQWBBTpk41oSZDv
+hSlsCLboVWzT5w414TAfBgNVHSMEGDAWgBTpk41oSZDvhSlsCLboVWzT5w414TAP
+BgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQBWeu3+kB2WmxXbIDPU
+1JGw4rvj/+u4mvN1maYkibqCZyKMXLuqwOy9wMhtniHgKp/RIxFI+W3FTq4Tik++
+kwDemaYq3nwbHMwBXwFh/T9I9ExtWBCogj+LLFUsrPDJNmUwYnnEMRh6beF8oT1E
+Da3oNVZ70Tyv0DnWozW+4TQZ8bTOQ/bpjoFNZPVB3Jr7tjVLfPez8m/clM8+War+
+gjTiiUsJkJP2uhKmWkb58CCiH+k2EH3fw2IUmc0fgTMGZ5vv8g1OjCBrXspnGSpk
+iJf9ryw/jIH/9RGxSUO7tiQxe/IShf65clsyxlAjrSr7JvbYwOyIXAbgNA7vk0lc
+nmmv
+-----END CERTIFICATE-----" > /var/opt/magma/certs/rootCA.pem
```

Comment:
What is that for a certificate? Where does it come from? How does one know, that it is trustworthy?
If it is essential for the workflow and not personalized, we should probably add it as file to the repository and not as code junk to the docs.

Reply:
The certificate here is just one I generated. I assume that someone that is deploying an AGW has the cert from their orc8r deployment. The reason this step is here, is that the [docker AGW script checks for it, and exits if it is not present.](https://github.com/magma/magma/blob/master/lte/gateway/deploy/agw_install_docker.sh#L34-L38)